src/Perpetuum/Services/Channels/ChatCommands/AdminCommandHandlers.cs

```diff
@@ -109,9 +109,9 @@ public static void CheckZoneId(AdminCommandData data, int zoneId)
 
         public static void CheckRequiredArgLength(AdminCommandData data, int expectedLength)
         {
-            if(data.Command.Args.Length != expectedLength)
+            if (data.Command.Args.Length != expectedLength)
             {
-                SendMessageToAll(data, $"Error - Command expects " + expectedLength.ToString() + " args, but was given " +data.Command.Args.Length.ToString()+ " args");
+                SendMessageToAll(data, $"Error - Command expects " + expectedLength.ToString() + " args, but was given " + data.Command.Args.Length.ToString() + " args");
```

Comment:
If you're using String Interpolation you don't need to use concatenation or `ToString()` conversion.

e.g. `$"Command expects {expectedLength} args, but was given {data.Command.Args.Length} args"` is fine